integration_tests/src/main/python/iceberg_test.py

```diff
@@ -154,7 +158,7 @@ def setup_iceberg_table(spark):
     ("uncompressed", None),
     ("snappy", None),
     ("gzip", None),
-    pytest.param(("lz4", "Unsupported compression type"),
+    pytest.param(("lz4", "Unsupported Parquet compression type"),
                  marks=pytest.mark.skipif(is_before_spark_320(),
```

Comment:
`is_before_spark_320` is useless now, because versions before 320 are not supported now.